plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/writer/S3TransactionLogSynchronizer.java

```diff
@@ -46,12 +48,12 @@
 import static java.util.Objects.requireNonNull;
 
 /**
- * The S3 Native synchronizer is a {@link TransactionLogSynchronizer} for S3 that requires no other dependencies.
+ * The S3 synchronizer is a {@link TransactionLogSynchronizer} for S3 that requires no other dependencies.
  */
-public class S3NativeTransactionLogSynchronizer
+public class S3TransactionLogSynchronizer
         implements TransactionLogSynchronizer
```

Comment:
would it be cleaner to have two implementations. And bind one or the other depending on exclusive writes suppported flag

Reply:
We could do that I guess